### ISSUE TITLE
`A` is not focusable in Firefox

### ISSUE THREAD

**Issue by andreasremdt** [solidjs/solid-router#234]
### Describe the bug

When using the `A` component from `solid-router`, I can't focus the rendered anchor element properly via keyboard. Pressing Tab jumps over the anchor as if it was non-focusable. **This issue only happens in Firefox**, not Chromium-based browsers.

### Your Example Website or App

https://stackblitz.com/edit/solidjs-templates-7joygr

### Steps to Reproduce the Bug or Issue

1. Open the example in Stackblitz in Firefox
2. Click into the first input on the page
3. Use your Tab key and try to focus the below anchor element

### Expected behavior

As a user, I expect to be able to focus all anchor elements rendered by `solid-router`, no matter the browser.

### Platform

- OS: macOS
- Browser: Firefox
- Version: 109

**Comment by Brian-Pob** [comment]
It looks like this is intended behavior for Firefox on macOS. https://stackoverflow.com/a/11713537

I have both a Windows 11 device and a MacBook Pro. I tested your Stackblitz with Firefox on both machines and found that the `A` tag is focuseable on Firefox on Windows but not on Mac. Below is a screen recording of my test.
https://watch.screencastify.com/v/iAW4GlP4y1yAmY8yY8Y3

After following the instructions on ths StackOverflow link, I was able to get `A` tag focusing to work.
